src/pages/home/components/MovieCard/MovieCard.tsx

```diff
@@ -1,15 +1,17 @@
 import React from "react";
 import './moviecard.css'
+import Kebab from "../Kebab/Kebab";
 
 export default function MovieCard(props: { img: string; title: string; year: string, description: string; }){
```

Comment:
You can get props like this `MovieCard({ img: string; title: string; year: string, description: string; }), and it would be better to extract component prop types a separate `type/interface`